src/components/twilio-voice-basic-call-control/routes.js

```diff
@@ -7,13 +7,13 @@ const { accountSid, apiKeySid, apiKeySecret } = config;
 const client = Twilio(apiKeySid, apiKeySecret, { accountSid });
 
 router.post('/conferences/:conferenceSid/participants/:callSid', async (req, res) => {
-  const { hold } = req.body;
+  const { hold, muted } = req.body;
```

Comment:
Since we can now pass hold or muted separately, the values might be undefined. We should not set an undefined value since it might cause unexpected backend behavior.